e2e/tests/upgrades/upgrade_test.go

```diff
@@ -223,7 +233,9 @@ func (s *UpgradeTestSuite) TestChainUpgrade() {
 	t := s.T()
 
 	ctx := context.Background()
-	chain := s.SetupSingleChain(ctx)
```

Comment:
I opted to remove this fn for now, and more native support for a single chain in a follow up (will create issues soon)

Reply:
I got you brother! :D https://github.com/cosmos/ibc-go/issues/3160